Content.Shared/_RMC14/Marines/Roles/Ranks/SharedRankSystem.cs

```diff
@@ -76,17 +76,18 @@ public void SetRank(EntityUid uid, ProtoId<RankPrototype> from)
     }
 
     /// <summary>
-    ///     Gets the rank name of a given mob.
+    /// Gets the rank name of a given mob.
     /// </summary>
     public string? GetRankString(EntityUid uid, bool isShort = false, bool hasPaygrade = false)
     {
         var rank = GetRank(uid);
-        if (rank == null)
+
+        if (rank is null)
```

Comment:
Same for this, we style most null checks with == and !=, not is and is not
```suggestion
        if (rank == null)
```